app/components/ui/Navigation.module.scss

```diff
@@ -172,3 +178,29 @@
     font-style: normal;
   }
 }
+
+.buttonLink {
+  appearance: button;
+  text-decoration: none;
+  color: white;
+  background-color: $color-brand;
+  border-radius: 3px;
+  font-size: 16px;
+  font-weight: 600;
+  height: 60px;
+  width: 200px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  &:focus,
+  &:hover {
+    color: white;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  }
+  &:active {
+    box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.15);
+  }
+  @media screen and (max-width: 600px) {
+    display: none;
+  }
+}
```

Comment:
This is somewhat copied/pasted from the search button, with a few tweaks. Not sure if we want to extract it to a shared style or not. Do we use this button styling elsewhere much?

Reply:
Yeah, I think it'd be good to extract this into a shared style, but it doesn't need to happen in this PR, since we're already duplicating it in several places, probably with subtle and possibly unintentional differences between them. Outside of the search button near the bottom of the home page, I know that it's also present on the Resource Guides (ServiceDiscoveryForm in the code) and the Edit Pages, and I'm sure we have buttons elsewhere too.

Reply:
I'll take a look at that!